CI/fix.grid.ui.ordering.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs").promises;
+const convert = require("xml-js");
+
+const ATTRIBUTE_KEY = "_attributes";
+const GRID_LAYOUT_CLASS = "QGridLayout";
+
+const uiDir = `${__dirname}/../src/ui`;
+
+function getCellOrder(node, attrKey) {
```

Comment:
Ah, I see how this is working now.
I can tell that it is just sorting the existing numbers into order - but it doesn't do the total clean-up that would also examine the `{row|col}span` numbers as well to work out if the numbering doesn't start at `0` or has any gaps that aren't covered by other cells in the same row or column overlapping because they occupy more than one row or column OR if more than one item occupies the same cell! OTOH the ordering that this does will make it fairly simple to clean up the other things without making too much git/diff noise.

😁 

I was confused at first when I looked at the most recent changes as it seemed to be making some `.ui`files worse - but then I *finally* realised that you were making things simpler by not running the clean-up code on the existing code-base... 🤣